packages/components/src/components/teaser/teaser.ts

```diff
@@ -0,0 +1,167 @@
+    super.updated(changedProperties);
+    if (changedProperties.has('breakpoint')) {
+      this.updateOrientation();
+    }
+  }
+
+  updateOrientation() {
+    this._orientation = this.offsetWidth >= this.breakpoint ? 'horizontal' : 'vertical';
+  }
+
+  render() {
+    if (this.variant === 'white border-neutral-300') {
+      this.inset = true;
+    }
+
+    const slots = {
+      'teaser-has-default': this.hasSlotController.test('[default]'),
+      'teaser-has-media': this.hasSlotController.test('media'),
+      'teaser-has-meta': this.hasSlotController.test('meta'),
+      'teaser-has-headline': this.hasSlotController.test('headline')
+    };
+
+    return html`
+      <div
+        class=${cx(
+          'flex',
+          {
+            white: 'bg-white',
+            'white border-neutral-300': 'bg-white border border-neutral-300',
+            'neutral-100': 'bg-neutral-100',
+            primary: 'bg-primary text-white',
+            'primary-100': 'bg-primary-100'
+          }[this.variant],
+          this._orientation === 'vertical' && 'flex-col',
+          this._orientation === 'horizontal' && 'flex-row gap-8',
+          this._orientation === 'horizontal' && this.inset && 'py-8 px-10'
+        )}
+        part="base"
+      >
+        <div
+          style=${this._orientation === 'horizontal' ? `width: var(--distribution-media, 100%);` : ''}
+          class=${cx(!this.inset && this._orientation === 'vertical' && 'mb-4', !slots['teaser-has-media'] && 'hidden')}
+          part="media"
+        >
+          <slot name="media"></slot>
+        </div>
+
+        <div
+          style=${this._orientation === 'horizontal'
+            ? `width: var(--distribution-content, 100%); ${
+                this.inset ? 'width: var(--distribution-content, calc(100% - 2rem));' : ''
+              }`
+            : ''}
+          class=${cx(
+            'flex flex-col text-left',
+            this._orientation === 'horizontal' && `flex flex-col`,
+            this._orientation === 'vertical' && this.inset && 'm-4'
+          )}
+          part="content"
+        >
+          <div part="meta" class=${cx('gap-2 mb-4', !slots['teaser-has-meta'] && 'hidden')}>
+            <slot name="meta"></slot>
+          </div>
+
+          <div
+            part="headline"
+            class=${cx('text-lg font-bold m-0', this.variant === 'primary' ? 'text-white' : 'text-black')}
+          >
+            <slot name="headline">Always insert one semantically correct heading element here (e. g. &lt;h2&gt;)</slot>
+          </div>
+
+          <div part="main" class=${cx(!slots['teaser-has-default'] && 'hidden')}>
+            <slot></slot>
+          </div>
+        </div>
+      </div>
+    `;
+  }
+
+  static styles = [
+    SolidElement.styles,
+    css`
+      :host {
+        display: block;
+      }
+
+      ::slotted(*) {
+        margin: 0;
+      }
+
+      ::slotted([slot='headline']) {
+        font-size: var(--sd-font-size-lg, 1.25rem) !important;
```

Comment:
Why do we need the !important here? Can't we get rid of that somehow? 🥲

Reply:
I know 🥲 Unfortunately without it it does not override the slotted headline styles, and we need it to look the same regardless of the <h> tag used in the slot.

Reply:
I think this is okay, as this should really be the size which design defined in the card. It shouldn't be changable.

Reply:
@mariohamann is this the expected behaviour or is something wrong with the order of overriding in our main file for CSS?

Reply:
Okay, your comment was not loaded. Good, than its okay for me.

Reply:
The problem is: We don't know how or even if people override/reset their CSS for headlines. And they really shouldn't have to care about when using the component.